src/backend/core/product/models.py

```diff
@@ -61,6 +61,20 @@ class Product(TranslatableModel):
     def __str__(self) -> str:
         return "id: {} title: {}".format(self.id, self.title)
 
+    def get_primary_photo(self):
+        from .models import (
+            ProductMedia,
+        )  # roductMediaTypes, )
```

Comment:
Can we delete `roductMediaTypes` comment please?

Reply:
Yeah, sure, I'll delete it soon. Sooorry

Reply:
@lakatop Remove